course3/homework-transcoder/test.cc

```diff
@@ -0,0 +1,17 @@
+#include "transcoder.h"
+
+#include <iostream>
+
+
+int main()
+{
+	std::cout << "Transcoding input.json to output.xml..." << std::endl;
```

Comment:
This is fine for a test driver for your library, but lets try an make it into a fully-usable, Unix-style tool.
This means that the following invocations should be possible:
```
$ transcoder --xml2json in.json out.xml
$ transcoder --help # prionts help and exists
$ transcoder -v --json2xml in.xml /dev/stdout # p-v prints extra debugging information to STDERR
```

Use Boost.ProgramOptions to parse command-line params